proto/cronos/tx.proto

```diff
@@ -3,11 +3,38 @@ package cryptoorgchain.cronos.cronos;
 
 // this line is used by starport scaffolding # proto/tx/import
 
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
 option go_package = "github.com/crypto-org-chain/cronos/x/cronos/types";
 
-// Msg defines the Msg service.
+// Msg defines the Cronos Msg service
 service Msg {
     // this line is used by starport scaffolding # proto/tx/rpc
+
+    // Send defines a method for converting ibc coins to Cronos coins.
+    rpc ConvertTokens(MsgConvertTokens) returns (MsgConvertResponse);
```

Comment:
I suppose we also need a message to convert erc20 back to native token and send through ibc channel?

Reply:
I think we can use "SendToCryptoOrg" message from smart contract 

For erc20: 
- ConvertToTokens will send ibc tokens to smart contract account and trigger mint erc20 in smart contract
- SendToCryptoOrg is trigger within smart contract is sending back ibc tokens from smart contract to user address and trigger IBC transfer

similarly to ethermint hook design

Reply:
I guess `SendToCryptoOrg` support any ibc channels, not limited to crypto-org then?
And is the `SendToCryptoOrg` triggered by smart contract using the evm hook proposal?